### ISSUE TITLE
question about learning_rate

### ISSUE THREAD

**Issue by Y-P-W** [bigmb/Unet-Segmentation-Pytorch-Nest-of-Unets#14]
In line 157th, MAX_STEP = int(1e10).
In CosineAnnealingLR(), T_Max is the number of epochs since the last restart in SGDR.
if you setup T_max = 1e10, the changes of Learning rate is very slow.  It needs to change it?
Also, why did you choose this CosineAnnealingLR()？
thanks!

**Comment by bigmb** [comment]
I chose CosineAnnealing due to some paper I was reading. It doesn't matter much in my case as its a binary segmentation.
I had a use batch size and less epochs so my iteration would end in 100 max.

You can change it but it does depend upon the dataset too.
https://pytorch.org/docs/stable/optim.html
I dont have much idea about your dataset but it will just help your model converge faster.
